tests/scripts/thread-cert/Cert_5_5_05_SplitMergeREED.py

```diff
@@ -175,11 +183,71 @@ def test(self):
         self.assertEqual(self.nodes[ROUTER1].get_state(), 'child')
         self.assertEqual(self.nodes[REED1].get_state(), 'router')
 
+        self.collect_ipaddrs()
+
         addrs = self.nodes[LEADER].get_addrs()
         for addr in addrs:
             if addr[0:4] != 'fe80':
                 self.assertTrue(self.nodes[ROUTER1].ping(addr))
 
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+
+        LEADER = pv.vars['LEADER']
+        ROUTER_1 = pv.vars['ROUTER_1']
+        ROUTER_2 = pv.vars['ROUTER_2']
+        ROUTER_3 = pv.vars['ROUTER_3']
+        REED = pv.vars['REED']
+        router1_pkts = pkts.filter_wpan_src64(ROUTER_1)
+        router2_pkts = pkts.filter_wpan_src64(ROUTER_2)
+        reed_pkts = pkts.filter_wpan_src64(REED)
+
+        router2_pkts.filter_mle_cmd(MLE_CHILD_ID_RESPONSE).must_next()
+        _start_idx = router2_pkts.index
+        _reed_pkts = reed_pkts.range(_start_idx)
```

Comment:
Can we just set the filter index?
```suggestion
        reed_pkts.index = _start_idx
```